runtests.py

```diff
@@ -826,8 +858,8 @@ def run_distutils(self, test_directory, module, workdir, incdir,
             ext_compile_flags = CFLAGS[:]
             compiler = COMPILER or sysconfig.get_config_var('CC')
 
-            if self.language == 'c' and compiler == 'gcc':
-                ext_compile_flags.extend(['-std=c89', '-pedantic'])
+            if self.language == 'c' and 'gcc' in compiler:
+                ext_compile_flags.append('-pedantic')
```

Comment:
I looks like because of the change I made here, although I removed the `-std=c89` flag, quite a few more warnings are being generated in the Travis-CI builds than there were previously due to the `-pedantic` flag, which was previously not being added due to the fact that `compiler != 'gcc'`.  I don't know if we particularly care about `-pedantic` or not--the `ISO C90 does not support ‘long long’ [-Wlong-long]` are especially annoying.